R/summary.R

```diff
@@ -221,13 +221,40 @@ param_estimates.bbi_nonmem_summary <- function(.summary) {
   num_methods <- length(.summary$parameters_data)
   param_names <- .summary$parameter_names
   param_estimates <- .summary$parameters_data[[num_methods]]$estimates
-  tibble::tibble(
+
+  param_df <- tibble::tibble(
     names = unlist(param_names),
     estimate = unlist(param_estimates),
     stderr = unlist(.summary$parameters_data[[num_methods]]$std_err) %||% NA_real_,
-    fixed = unlist(.summary$parameters_data[[num_methods]]$fixed),
+    random_effect_sd = c(
+      rep(NA, length(param_names$theta)),
+      unlist(.summary$parameters_data[[num_methods]]$random_effect_sd)),
```

Comment:
These pieces likely need some `%||% NA_real_` portions as this `.summary$parameters_data[[num_methods]]$random_effect_sd` could be NULL